modules/gitops-release-notes-1-16-5.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-16.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-16-5_{context}"]
+= Release notes for {gitops-title} 1.16.5
+
+{gitops-title} 1.16.5 is now available on {OCP} 4.12, 4.14, 4.15, 4.16, 4.17, and 4.18.
+
+[IMPORTANT]
+====
+After completing the upgrade, run the `audit-operator-roles.sh` audit script to review namespace-scoped access created by the {gitops-title} Operator. The script identifies `Role` and `RoleBinding` objects that grant cross-namespace access when the _Applications in any namespace_ feature or _ApplicationSets in any namespace_ feature is enabled using the Argo CD custom resource (CR) (`.spec.sourceNamespaces` and `.spec.applicationSet.sourceNamespaces`). These features are disabled by default. The script scans all namespaces and reports `Role`/`RoleBinding` combinations that meet the following conditions:
+
+* Grant access to the `argoproj.io/Application` resource
+* Are labeled with `app.kubernetes.io/part-of=argocd`
+* Bind to a service account in a different namespace, resulting in cross-namespace access
+
+Review the script output to ensure that cross-namespace access is limited to the intended namespaces only. 
+
+For more information about the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/audit-operator-roles.sh[Audit Operator script]. 
+
+For more information about the steps to execute the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/README.md[Audit Operator script readme].
```

Comment:
As discussed, it is okay to put external links if it is absolutely necessary. Since this link is to upstream docs and the information and content is important, I also recommend bringing the content downstream in the future. :)